cache/remotecache/s3/s3.go

```diff
@@ -360,6 +360,13 @@ func newS3Client(ctx context.Context, config Config) (*s3Client, error) {
 	client := s3.NewFromConfig(cfg, func(options *s3.Options) {
 		if config.AccessKeyID != "" && config.SecretAccessKey != "" {
 			options.Credentials = credentials.NewStaticCredentialsProvider(config.AccessKeyID, config.SecretAccessKey, config.SessionToken)
+		} else {
+      // Anonymous credentials
+      // Citing SDK documentation:
+      // "If using the `NewFromConfig` constructor you'll need to explicitly set
+      //  the `Credentials` member to nil, if the external config resolved a
+      //  credential provider."
+			options.Credentials = nil
```

Comment:
Can you fix the indentation of this comment and also add a link to the SDK docs. Thanks

Reply:
@crazy-max I've updated the documentation and added a link to the documentation instead of the quote.